### PR TITLE
Dave add new node attributes

### PR DESCRIPTION
**Pull request by DavidDeutz**
Closes *list issues numbers here*

### Developer Checklist (Definition of Done)

**Issue**

- [ ] All acceptance criteria from the issue are met
- [ ] Tested in latest Chrome/Firefox

**UI/UX/Vis**

- [ ] Requires UI/UX/Vis review
  - [ ] Reviewer(s) are notified (_tag assignees_)
  - [ ] Review has occurred (_link to notes_)
  - [ ] Feedback is included in this PR
  - [ ] Reviewer(s) approve of concept and design

**Code**

- [ ] Branch is up-to-date with the branch to be merged with, i.e., develop
- [ ] Code is cleaned up and formatted
- [ ] Unit tests are written (frontend/backend if applicable)
- [ ] Integration tests are written (if applicable)

**PR**

- [x] Descriptive title for this pull request is provided (will be used for release notes later)
- [x] Reviewer and assignees are defined
- [x] Add type label (e.g., *bug*, *feature*) to this pull request
- [ ] Add release label (e.g., `release: minor`) to this PR following [semver](https://semver.org/)
- [ ] The PR is connected to the corresponding issue (via `Closes #...`)
- [ ] [Summary of changes](#summary-of-changes) is written

### Summary of changes
Added new version of tree view
- 

### Additional notes for the reviewer(s)

-  
Thanks for creating this pull request 🤗

### CODE REVIEW

File: src/onNodesVisibilityChange.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const onNodesVisibilityChange = (reactflow, nodes, visible)=> {
         
         const image = node.hidden ? hiddenIcon : shownIcon
 
-        document.getElementById(node.id + "_icon").setAttribute('src', image)
+        //document.getElementById(node.id + "_icon").setAttribute('src', image)
```
Comment (Samiwaa) [inline src/onNodesVisibilityChange.tsx:29]: if not needed please delete

File: src/SidebarFilterList.tsx [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+import { List, Card, Text, ScrollArea, ThemeIcon, Checkbox, Group, Select, Divider } from "@mantine/core"
 import { onNodesVisibilityChange } from './onNodesVisibilityChange';
 
+import { TreeView } from '@mui/x-tree-view';
+import { TreeItem } from '@mui/x-tree-view';
+import type {} from '@mui/x-tree-view/themeAugmentation';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
 import shownIcon from './images/eye.png'
```
Comment (Samiwaa) [inline src/SidebarFilterList.tsx]: Please remove unused imports

File: src/onNodesVisibilityChange.tsx (hunk 1 above)
Comment (DavidDeutz) [reply src/onNodesVisibilityChange.tsx:29]: > if not needed please delete

was commented out intentionally for further use in future when implementing the icons with hide onClick event in future

File: src/SidebarFilterList.tsx (hunk 2 above)
Comment (DavidDeutz) [reply src/SidebarFilterList.tsx]: > Please remove unused imports

done

**Review by Samiwaa — APPROVED** [review]

**Pull request merged**